src/parameters.toml

```diff
@@ -1641,10 +1641,14 @@ type = "float"
 value = 273.15
 type = "float"
 
-[temperature_saturation_adjustment_min]
+[temperature_saturation_adjustment_init_min]
 value = 150
 type = "float"
 
+[temperature_saturation_adjustment_min]
+value = 1
```

Comment:
I vote for 0.42 ;)

Reply:
I'm fine with either one, but I wouldn't be surprised if we're scratching our heads down the road either way :)